cli/pkg/kctrl/cmd/package/installed/create_or_update.go

```diff
@@ -299,17 +303,38 @@ func (o CreateOrUpdateOptions) update(client kubernetes.Interface, kcClient kccl
 		return err
 	}
 
-	if o.valuesFile == "" && !changed {
-		return err
+	if o.valuesFile == "" && !changed && o.values {
+		o.statusUI.PrintMessagef("No changes to package install '%s' in namespace '%s'", o.Name, o.NamespaceFlags.Name)
+		return nil
 	}
 
 	isSecretCreated, err := o.createOrUpdateValuesSecret(updatedPkgInstall, client)
 	if err != nil {
+		o.statusUI.PrintMessagef("", o.Name)
 		return err
 	}
 
-	o.statusUI.PrintMessagef("Updating package install for '%s'", o.Name)
-	o.addCreatedResourceAnnotations(&updatedPkgInstall.ObjectMeta, false, isSecretCreated)
+	isSecretDeleted := false
+	if !o.values && len(updatedPkgInstall.Spec.Values) > 0 {
+		isSecretDeleted, err = o.dropValuesSecret(client)
+		if err != nil {
+			return err
```

Comment:
Should we have something like this instead of wrapping the error at all the return statements in `dropValuesSecret`?
```suggestion
			fmt.Errorf("Deleting values secret: %s", err.Error())
```

Reply:
Yep makes sense! did that